Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -789,6 +785,9 @@ public function testAnnotationsBundleMappingDetection(): void
         ]);
     }
 
+    // Disabled to prevent changing the comment below to a single-line annotation
+    // phpcs:disable SlevomatCodingStandard.Commenting.RequireOneLineDocComment.MultiLineDocComment
```

Comment:
better to put exclude rule in phpcs.xml.dist, or to use if PHP_VERSION_ID + $this->skip, wouldn't it?